### PR TITLE
docs(optimize): add tile copying

### PR DESCRIPTION
**Pull request by misiekhardcore**
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: optimize/components/userguide/edit-mode.md [hunk 1]
```diff
@@ -34,7 +34,11 @@ In the **Add a Tile** modal, choose **Text** to access the text editor. This ena
 
 ![text editor](./img/dashboard-addAReportModal-textReport.png)
 
-### Placing tile on the dashboard
+## Copying, Editing, and Removing Tiles
```
Comment (christinaausley) [inline optimize/components/userguide/edit-mode.md:37]: Grammatical note to keep this sentence case, so "Copying, editing, and removing titles"

File: optimize/components/userguide/edit-mode.md [hunk 2]
```diff
@@ -34,7 +34,11 @@ In the **Add a Tile** modal, choose **Text** to access the text editor. This ena
...
+
+Each tile features a set of buttons in its top-right corner, providing you with options to edit and remove the tile. The 'Copy' button allows you to duplicate any tile, and the copied version can be conveniently placed anywhere on the dashboard. This duplicated tile is then customizable, enabling you to tailor it to your specific requirements.
```
Comment (christinaausley) [inline optimize/components/userguide/edit-mode.md:39]: I would say the "copy icon" here. You could even add a microimage of this, for example `Navigation to Web Modeler in the global nav`

File: optimize/components/userguide/img/dashboard-reportEditActions.png
Comment (christinaausley) [inline optimize/components/userguide/img/dashboard-reportEditActions.png:1]: I don't love that the arrows partially cover what we are pointing to, can we tighten these arrows up or just add microimages of the icons where appropriate?

If we keep this screenshot, we should just ensure we have a period after each sentence like we do in a few of them for consistency.

**Review by christinaausley — COMMENTED** [review]
A few comments to clean up 👍

File: optimize/components/userguide/edit-mode.md (hunk 1 above)
Comment (misiekhardcore) [reply optimize/components/userguide/edit-mode.md:37]: oh, sorry, my bad

File: optimize/components/userguide/edit-mode.md (hunk 2 above)
Comment (misiekhardcore) [reply optimize/components/userguide/edit-mode.md:39]: I like the idea of microimages in the text. I didn't know we can do this. I will give it a try ;)

File: optimize/components/userguide/img/dashboard-reportEditActions.png
Comment (misiekhardcore) [reply optimize/components/userguide/img/dashboard-reportEditActions.png:1]: I don't love it either but I am not not sure if I can make it look better as these annotations are generated by our e2e tests. Will it be ok to leave it like it is with the consisten textx and adding the microimages in the text?

Comment (misiekhardcore) [reply optimize/components/userguide/img/dashboard-reportEditActions.png:1]: I moved those annotations around, maybe it's a bit better now?

**Review by christinaausley — APPROVED** [review]
Nice adjustments!

**Review by OmranAbazid — APPROVED** [review]

**Pull request merged**